src/test/scala/net/ripe/rpki/publicationserver/BinariesTest.scala

```diff
@@ -12,6 +14,12 @@ class BinariesTest extends FunSuite with Matchers {
     check(Base64("DEADBEEF"))
   }
 
-  private def check(base64: Base64): Assertion =
+  private def check(base64: Base64): Assertion = {
     Bytes.toBase64(Bytes.fromBase64(base64)) should be(base64)
+
+//    val baos = new ByteArrayOutputStream();
```

Comment:
Left by accident?

Reply:
Yeah, good point, it was some scaffolding left unused.